operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1284,5 +763,516 @@ spec:
+            - routes
+            - routes/custom-host
+          verbs:
+            - get
+            - list
+            - watch
+            - create
+            - delete
+            - patch
+            - update
+        - apiGroups:
+            - policy
+          resources:
+            # The cluster operator needs to access and manage pod disruption budgets this limits the number of concurrent disruptions
+            # that a Strimzi component experiences, allowing for higher availability
+            - poddisruptionbudgets
+          verbs:
+            - get
+            - list
+            - watch
+            - create
+            - delete
+            - patch
+            - update
+        serviceAccountName: strimzi-cluster-operator
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - "rbac.authorization.k8s.io"
+          resources:
+          # The cluster operator needs to create and manage cluster role bindings in the case of an install where a user
+          # has specified they want their cluster role bindings generated
+          - clusterrolebindings
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+          - patch
+          - update
+        - apiGroups:
+          - storage.k8s.io
+          resources:
+          # The cluster operator requires "get" permissions to view storage class details
+          # This is because only a persistent volume of a supported storage class type can be resized
+          - storageclasses
+          verbs:
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          # The cluster operator requires "list" permissions to view all nodes in a cluster
+          # The listing is used to determine the node addresses when NodePort access is configured
+          # These addresses are then exposed in the custom resource states
+          # The Kafka Brokers require "get" permissions to view the node they are on
+          # This information is used to generate a Rack ID that is used for High Availability configurations
+          - nodes
+          verbs:
+          - get
+          - list
+        serviceAccountName: strimzi-cluster-operator
+      deployments:
+      - name: amq-streams-cluster-operator-v1.7.0
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: strimzi-cluster-operator
+              strimzi.io/kind: cluster-operator
+          template:
+            metadata:
+              labels:
+                name: strimzi-cluster-operator
+                strimzi.io/kind: cluster-operator
+            spec:
+              serviceAccountName: strimzi-cluster-operator
+              volumes:
+                - name: strimzi-tmp
+                  emptyDir:
+                    medium: Memory
+                - name: co-config-volume
+                  configMap:
+                    name: strimzi-cluster-operator
```

Comment:
You seem to have changed the name. Which name is used in the docs? Is the change worth the possible confusion for users?

Reply:
I changed it back to the original name for 1.7.0 as I don't know what the impact would be for changing it. I'll look into aligning it with upstream next release